web/src/admin/users/UserListPage.ts

```diff
@@ -392,27 +394,24 @@ export class UserListPage extends WithBrandConfig(WithCapabilitiesConfig(TablePa
     renderObjectCreate(): TemplateResult {
         return html`
             <ak-forms-modal>
-                <span slot="submit">${msg("Create")}</span>
-                <span slot="header">${msg("Create User")}</span>
+                <span slot="submit">${msg("Create User")}</span>
+                <span slot="header">${msg("New User")}</span>
```

Comment:
Planning for this to be a gradual change as test coverage expands:

- "New" as prefix for buttons which open a form or wizard.
- "Create" as a prefix for buttons which perform the submission